arm/Microsoft.RecoveryServices/vaults/.parameters/min.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "sxx-az-rsv-x-001"
```

Comment:
I'd suggest 002 for min

Reply:
Updted to 002 & the other parameter file to 001